schema/controller/deployment.json

```diff
@@ -45,6 +45,10 @@
     "deploy_timeout": {
       "$ref": "/schema/controller/common#/definitions/deploy_timeout"
     },
+    "batch_size": {
```

Comment:
I think this should be called `deploy_batch_size` here and in the database schema.

Reply:
Updated in d9abd6ab